src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/cas2/ApplicationsTransformerTest.kt

```diff
@@ -85,6 +91,33 @@ class ApplicationsTransformerTest {
       assertThat(result.status).isEqualTo(ApplicationStatus.submitted)
       assertThat(result.telephoneNumber).isEqualTo(application.telephoneNumber)
     }
+
+    @Test
+    fun `transformJpaToApi transforms a submitted CAS2 application correctly with status updates`() {
+      val statusUpdateEntity = Cas2StatusUpdateEntityFactory().withLabel("status update")
+        .withApplication(submittedCas2ApplicationFactory.produce()).produce()
+      val statusUpdateForApi = Cas2StatusUpdate(
+        id = statusUpdateEntity.id,
+        name = "statusUpdate",
+        label = "status update",
+        description = "status update",
+      )
+      every { mockStatusUpdateTransformer.transformJpaToApi(statusUpdateEntity) } returns statusUpdateForApi
+
+      val application = submittedCas2ApplicationFactory.withStatusUpdates(
+        mutableListOf(
+          statusUpdateEntity,
+        ),
+      )
+        .produce()
+
+      val result = applicationsTransformer.transformJpaToApi(application, mockk())
+
+      assertThat(result.id).isEqualTo(application.id)
```

Comment:
A minor idea that we could ignore for now: if this test is all about status updates I don't think we get value by repeating assertions covered elsewhere. There's a cost (that is currently small) where should we need to change those assertions later we'd need to involve ourselves in potentially changing many other tests.

Reply:
makes sense. as I had to rebase anyway I went in and removed check for `status` and `telephoneNumber`